contracts/migrations/4_user_replica_set_migration.js

```diff
@@ -47,32 +47,40 @@ module.exports = (deployer, network, accounts) => {
             'address',
             'bytes32',
             'address',
-            'uint[]',
-            'address[]',
             'uint'
         ],
         [
           registryAddress,
           userFactoryKey,
           userReplicaSetBootstrapAddress,
-          bootstrapSPIds,
-          bootstrapNodeDelegateWallets,
           networkId
         ]
     )
     // Deploy proxy contract with encoded initialize function
     let deployedProxyTx = await deployer.deploy(
-      AudiusAdminUpgradeabilityProxy2,
+      AdminUpgradeabilityProxy,
       logicContractAddress,
       proxyAdminAddress,
       initializeUserReplicaSetManagerCalldata
     )
     let userReplicaSetManagerProxyAddress = deployedProxyTx.address
     console.log(`UserReplicaSetManager Proxy Contract deployed at ${deployedProxyTx.address}`)
 
+    // Confirm seed is not yet complete
+    let userReplicaSetManagerInst = await UserReplicaSetManager.at(deployedProxyTx.address)
+    let seedComplete = await userReplicaSetManagerInst.getSeedComplete({ from: userReplicaSetBootstrapAddress })
+    console.log(`Seed complete: ${seedComplete}`)
```

Comment:
do these checks + logs need to be merged?

Reply:
why would they not be merged? since we won't be enabling this as part of local setup